src/plugins/grass/qtermwidget/Screen.cpp

```diff
@@ -367,10 +367,10 @@ void Screen::setDefaultMargins()
 
    Note that RE_BOLD is used in both widget
    and screen rendition. Since xterm/vt102
-   is to poor to distinguish between bold
+   is too poor to distinguish between bold
    (which is a font attribute) and intensive
    (which is a color attribute), we translate
-   this and RE_BOLD in falls eventually appart
+   this and RE_BOLD in falls eventually separate
```

Comment:
this sentence doesn't make much sense. Please review this

Reply:
qtermwidget is a 3rd party library we've just imported direct. I wouldn't be concerned with these strings given they they'll just be reset when this internal copy of the library is next re-synced with upstream.

Reply:
removed